core/esmf-aspect-model-java-generator/src/main/java/org/eclipse/esmf/aspectmodel/java/JavaCodeGenerationConfig.java

```diff
@@ -29,7 +29,10 @@ public record JavaCodeGenerationConfig(
       String packageName,
       ImportTracker importTracker,
       boolean executeLibraryMacros,
-      File templateLibFile
+      File templateLibFile,
+      String aspectPrefix,
+      String aspectPostfix
```

Comment:
Can we rename this to e.g. "namePrefix" and "namePostfix" (since it also applies to Entities and AbstractEntities)?